src/main/java/delete_task_use_case/DeleteTaskInteractor.java

```diff
@@ -9,7 +9,8 @@ public class DeleteTaskInteractor implements DeleteTaskInputBoundary {
 
     private final DeleteTaskOutputBoundary outputBoundary;
     private final DeleteTaskDsGateway dsGateway;
-    private final String[] DAYSOFWEEK = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
+
+    private final String[] WEEK = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
```

Comment:
Renaming this to WEEK could imply association to the week class, which is not the case for this object. If we must rename it, we should call it DAYS_OF_WEEK. This applies to all instances of this change.